.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,68 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - kinetic-devel
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - ROS_DISTRO: kinetic
+            ROS_REPO: main
+          - ROS_DISTRO: melodic
+            ROS_REPO: main
+          - ROS_DISTRO: noetic
+            ROS_REPO: main
+          - ROS_DISTRO: noetic
+            ROS_REPO: testing
```

Comment:
Do we need noetic and melodic here? Since this CI will only run kinetic-devel I think we would only need kinetic env.

Reply:
This branch works for all three (and is released for all three).

Reply:
Oh then it is appropriate. Thanks for the PR!